pkg/datapath/config.go

```diff
@@ -123,14 +123,21 @@ type ConfigWriter interface {
 	WriteEndpointConfig(w io.Writer, cfg EndpointConfiguration) error
 }
 
-// RemoteSNATDstAddrExclusionCIDR returns a CIDR for SNAT exclusion. Any
+// RemoteSNATDstAddrExclusionCIDRv4 returns a CIDR for SNAT exclusion. Any
 // packet sent from a local endpoint to an IP address belonging to the CIDR
 // should not be SNAT'd.
-func RemoteSNATDstAddrExclusionCIDR() *cidr.CIDR {
+func RemoteSNATDstAddrExclusionCIDRv4() *cidr.CIDR {
 	if c := option.Config.IPv4NativeRoutingCIDR(); c != nil {
 		// native-routing-cidr is set, so use it
 		return c
 	}
 
 	return node.GetIPv4AllocRange()
 }
+
+// RemoteSNATDstAddrExclusionCIDRv6 returns a IPv6 CIDR for SNAT exclusion. Any
+// packet sent from a local endpoint to an IP address belonging to the CIDR
+// should not be SNAT'd.
+func RemoteSNATDstAddrExclusionCIDRv6() *cidr.CIDR {
+	return node.GetIPv6AllocRange()
```

Comment:
Is this pod CIDR node-local or cluster-wide?

Reply:
This is the node-local IPv6 allocation range.